edgraph/server.go

```diff
@@ -287,7 +287,15 @@ func parseSchemaFromAlterOperation(ctx context.Context, op *api.Operation) (*sch
 		return nil, err
 	}
 
+	preds := make(map[string]struct{})
```

Comment:
Note to reviewers mystified by this, `map[string]struct{}` is poor-man's _set_ in golang